### ISSUE TITLE
Newest Update broke the Auto Tagger

### ISSUE THREAD

**Issue by TheRamosOnline** [starik222/BooruDatasetTagManager#163]
I'm getting the following error while trying to launch the auto-tagger;

```
Traceback (most recent call last):
  File "D:\AI Art\Tagger\interrogator_rpc\main.py", line 31, in <module>
    import rpc_proto.services_pb2
  File "D:\AI Art\Tagger\interrogator_rpc\rpc_proto\services_pb2.py", line 9, in <module>
    from google.protobuf import runtime_version as _runtime_version
ImportError: cannot import name 'runtime_version' from 'google.protobuf' (C:\Users\XXXX\AppData\Local\Programs\Python\Python310\lib\site-packages\google\protobuf\__init__.py)
```

I have no idea how to fix this, I've tried reinstalling, updating protobuf and even ran it through ChatGPT to see if it could fix the issue, but nothing.

**Comment by starik222** [comment]
Please write the version of python and installed protobuf:
`python --version`
`pip show protobuf`

**Comment by TheRamosOnline** [comment]
C:\Users\FatMa>python --version
Python 3.10.6

C:\Users\FatMa>pip show protobuf
Name: protobuf
Version: 5.28.2
Summary:
Home-page: https://developers.google.com/protocol-buffers/
Author: protobuf@googlegroups.com
Author-email: protobuf@googlegroups.com
License: 3-Clause BSD License
Location: c:\users\fatma\anaconda3\lib\site-packages
Requires:
Required-by: grpcio-tools, onnx, onnxruntime-gpu, tb-nightly

**Comment by starik222** [comment]
This may be because you are using a `base` anaconda environment and there are some conflicts. Try the following:
Copy [this](https://github.com/starik222/BooruDatasetTagManager/raw/refs/heads/master/interrogator_rpc/requirements.txt) file to the interrogator_rpc folder (With replacement of the existing file)
In the anaconda console, create a new environment and install the dependencies.
`conda create -n bdtm python=3.10.13`
`conda activate bdtm`
`conda install conda-forge::vs2015_runtime`
Go to the interrogator_rpc directory
`pip install -r requirements.txt`
`python main.py`
I tested the installation in the new environment and everything worked successfully.

**Comment by TheRamosOnline** [comment]
```
Installing collected packages: pytz, mpmath, flatbuffers, urllib3, tzdata, typing-extensions, sympy, six, safetensors, regex, pyyaml, pyreadline3, protobuf, pillow, packaging, numpy, networkx, MarkupSafe, idna, grpcio, fsspec, filelock, einops, colorama, charset-normalizer, certifi, tqdm, requests, python-dateutil, onnx, jinja2, humanfriendly, grpcio-tools, bottleneck, torch, pandas, huggingface_hub, coloredlogs, torchvision, torchaudio, tokenizers, onnxruntime-gpu, transformers, timm
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
basicsr 1.4.2 requires addict, which is not installed.
basicsr 1.4.2 requires lmdb, which is not installed.
basicsr 1.4.2 requires opencv-python, which is not installed.
blendmodes 2022 requires aenum<4,>=3.1.7, which is not installed.
facexlib 0.3.0 requires opencv-python, which is not installed.
filterpy 1.4.5 requires matplotlib, which is not installed.
gfpgan 1.3.8 requires lmdb, which is not installed.
gfpgan 1.3.8 requires opencv-python, which is not installed.
realesrgan 0.3.0 requires opencv-python, which is not installed.
tb-nightly 2.14.0a20230623 requires absl-py>=0.4, which is not installed.
tb-nightly 2.14.0a20230623 requires google-auth<3,>=1.6.3, which is not installed.
tb-nightly 2.14.0a20230623 requires google-auth-oauthlib<1.1,>=0.5, which is not installed.
tb-nightly 2.14.0a20230623 requires markdown>=2.6.8, which is not installed.
tb-nightly 2.14.0a20230623 requires tensorboard-data-server<0.8.0,>=0.7.0, which is not installed.
tb-nightly 2.14.0a20230623 requires werkzeug>=1.0.1, which is not installed.
torchsde 0.2.5 requires boltons>=20.2.1, which is not installed.
torchsde 0.2.5 requires trampoline>=0.1.2, which is not installed.
blendmodes 2022 requires numpy<2,>=1.22.1, but you have numpy 2.1.2 which is incompatible.
blendmodes 2022 requires Pillow<10,>=9.0.0, but you have pillow 11.0.0 which is incompatible.
numba 0.57.1 requires numpy<1.25,>=1.21, but you have numpy 2.1.2 which is incompatible.
scipy 1.10.1 requires numpy<1.27.0,>=1.19.5, but you have numpy 2.1.2 which is incompatible.
Successfully installed MarkupSafe-3.0.1 bottleneck-1.4.1 certifi-2024.8.30 charset-normalizer-3.4.0 colorama-0.4.6 coloredlogs-15.0.1 einops-0.8.0 filelock-3.16.1 flatbuffers-24.3.25 fsspec-2024.9.0 grpcio-1.66.2 grpcio-tools-1.66.2 huggingface_hub-0.26.0 humanfriendly-10.0 idna-3.10 jinja2-3.1.4 mpmath-1.3.0 networkx-3.4.1 numpy-2.1.2 onnx-1.17.0 onnxruntime-gpu-1.19.2 packaging-24.1 pandas-2.2.3 pillow-11.0.0 protobuf-5.27.2 pyreadline3-3.5.4 python-dateutil-2.9.0.post0 pytz-2024.2 pyyaml-6.0.2 regex-2024.9.11 requests-2.32.3 safetensors-0.4.5 six-1.16.0 sympy-1.13.1 timm-1.0.11 tokenizers-0.20.1 torch-2.5.0+cu118 torchaudio-2.5.0+cu118 torchvision-0.20.0+cu118 tqdm-4.66.5 transformers-4.46.0.dev0 typing-extensions-4.12.2 tzdata-2024.2 urllib3-2.2.3

[notice] A new release of pip is available: 23.3.2 -> 24.2
[notice] To update, run: python.exe -m pip install --upgrade pip
(bdtm) PS D:\AI Art\Tagger\interrogator_rpc> python main.py
Have CUDA:  True
CUDA Count:  1
Using interrogator device: cuda
C:\Users\FatMa\AppData\Roaming\Python\Python310\site-packages\scipy\__init__.py:160: UserWarning: A NumPy version >=1.19.5 and <1.27.0 is required for this version of SciPy (detected version 2.1.2)
  warnings.warn(f"A NumPy version >={np_minversion} and <{np_maxversion}"
Traceback (most recent call last):
  File "C:\Users\FatMa\anaconda3\envs\bdtm\lib\site-packages\transformers\utils\import_utils.py", line 1770, in _get_module
    return importlib.import_module("." + module_name, self.__name__)
  File "C:\Users\FatMa\anaconda3\envs\bdtm\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\FatMa\anaconda3\envs\bdtm\lib\site-packages\transformers\models\blip\modeling_blip.py", line 29, in <module>
    from ...modeling_utils import PreTrainedModel
  File "C:\Users\FatMa\anaconda3\envs\bdtm\lib\site-packages\transformers\modeling_utils.py", line 48, in <module>
    from .loss.loss_utils import LOSS_MAPPING
  File "C:\Users\FatMa\anaconda3\envs\bdtm\lib\site-packages\transformers\loss\loss_utils.py", line 19, in <module>
    from .loss_deformable_detr import DeformableDetrForObjectDetectionLoss, DeformableDetrForSegmentationLoss
  File "C:\Users\FatMa\anaconda3\envs\bdtm\lib\site-packages\transformers\loss\loss_deformable_detr.py", line 6, in <module>
    from .loss_for_object_detection import (
  File "C:\Users\FatMa\anaconda3\envs\bdtm\lib\site-packages\transformers\loss\loss_for_object_detection.py", line 28, in <module>
    from scipy.optimize import linear_sum_assignment
  File "C:\Users\FatMa\AppData\Roaming\Python\Python310\site-packages\scipy\optimize\__init__.py", line 404, in <module>
    from ._optimize import *
  File "C:\Users\FatMa\AppData\Roaming\Python\Python310\site-packages\scipy\optimize\_optimize.py", line 30, in <module>
    from numpy import (atleast_1d, eye, argmin, zeros, shape, squeeze,
ImportError: cannot import name 'Inf' from 'numpy' (C:\Users\FatMa\anaconda3\envs\bdtm\lib\site-packages\numpy\__init__.py)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\AI Art\Tagger\interrogator_rpc\main.py", line 29, in <module>
    import interrogator
  File "D:\AI Art\Tagger\interrogator_rpc\interrogator.py", line 13, in <module>
    from ext_kohya import (
  File "D:\AI Art\Tagger\interrogator_rpc\ext_kohya\tagger.py", line 8, in <module>
    from .interrogators import WaifuDiffusionTagger, DepDanbooruTagger
  File "D:\AI Art\Tagger\interrogator_rpc\ext_kohya\interrogators\__init__.py", line 1, in <module>
    from .blip_large_captioning import BLIPLargeCaptioning
  File "D:\AI Art\Tagger\interrogator_rpc\ext_kohya\interrogators\blip_large_captioning.py", line 3, in <module>
    from transformers import BlipProcessor, BlipForConditionalGeneration
  File "<frozen importlib._bootstrap>", line 1075, in _handle_fromlist
  File "C:\Users\FatMa\anaconda3\envs\bdtm\lib\site-packages\transformers\utils\import_utils.py", line 1759, in __getattr__
    value = getattr(module, name)
  File "C:\Users\FatMa\anaconda3\envs\bdtm\lib\site-packages\transformers\utils\import_utils.py", line 1758, in __getattr__
    module = self._get_module(self._class_to_module[name])
  File "C:\Users\FatMa\anaconda3\envs\bdtm\lib\site-packages\transformers\utils\import_utils.py", line 1772, in _get_module
    raise RuntimeError(
RuntimeError: Failed to import transformers.models.blip.modeling_blip because of the following error (look up to see its traceback):
cannot import name 'Inf' from 'numpy' (C:\Users\FatMa\anaconda3\envs\bdtm\lib\site-packages\numpy\__init__.py)
(bdtm) PS D:\AI Art\Tagger\interrogator_rpc>
```

I'm not sure if I did something wrong?
